src/software/jetson_nano/setup_nano.sh

```diff
@@ -34,7 +34,7 @@ pip_libaries=(
     adafruit-circuitpython-rgb-display==3.10.9
     Adafruit-PlatformDetect==3.18.0
     Adafruit-PureIO==1.1.9
-    Jetson.GPIO==2.0.17
+    Jetson.Gpio==2.0.17
```

Comment:
There are various changes with the casing of GPIO in this PR that should not be here. I think they're carried on from an old branch. This one specifically should not be here as it is a package name and the casing matters

Reply:
bump @suchirss

Reply:
fixed. I'm assuming this is the one instance of GPIO left to fix you were talking about @itsarune